foreshadow/core/fs_logging.py

```diff
@@ -0,0 +1,264 @@
+"""Logging methods interfacing with a potential GUI system."""
```

Comment:
One request would be to add logging a small, newer component of foreshadow so that can be used as a reference for future.

Reply:
I was planning to add logging to anything we write from now on so those would all be references?